apps/redi-talent-pool/src/components/organisms/ExpiredJobListings.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  TpJobListingStatus,
+  useTpJobListingPatchMutation,
+} from '@talent-connect/data-access'
+import { Button } from '@talent-connect/shared-atomic-design-components'
+import { formatDistance } from 'date-fns'
+import { Columns, Element } from 'react-bulma-components'
+import { useQueryClient } from 'react-query'
+import { EmptySectionPlaceholder } from '../molecules/EmptySectionPlaceholder'
+import JobPlaceholderCardUrl from './company-profile-editables/job-placeholder-card.svg'
+import { JobListingCard } from './JobListingCard'
+
+export function ExpiredJobListings({ jobListings }) {
+  const updateMutation = useTpJobListingPatchMutation()
+  const queryClient = useQueryClient()
+
+  const onReactivateCTAClick = (joblistingId) => {
+    updateMutation.mutate(
+      {
+        input: {
+          id: joblistingId,
+          status: TpJobListingStatus.Active,
+          expiresAt: new Date().setMonth(new Date().getMonth() + 6),
+        },
+      },
+      {
+        onSuccess: () => {
+          queryClient.invalidateQueries()
+        },
+      }
+    )
+  }
+
+  const isEmpty = jobListings?.length === 0
+
+  const renderTimestamp = (expiresAt) =>
+    `Expired ${formatDistance(new Date(expiresAt), new Date(), {
+      addSuffix: true,
+    })}`
+
+  const renderReactivateCTA = (jobListingId) => {
+    return (
+      <Button simple onClick={() => onReactivateCTAClick(jobListingId)}>
+        Reactivate
+      </Button>
+    )
+  }
+
+  return (
```

Comment:
You can simplify this by moving the positive-case return into a `if (isEmpty) return <blabla when empty>` up here.

Reply:
I'm not sure I get this one. We should have the title with the underline, no mater if the section is empty or not.